vendors/cypress/boards/CY8CKIT_064S0S2_4343W/aws_demos/config_files/aws_demo_config.h

```diff
@@ -30,6 +30,7 @@
  * Only one demo can be configured at a time
  *
  *          CONFIG_MQTT_DEMO_ENABLED
+ *          CONFIG_CSDK_DEVICE_SHADOW_DEMO_ENABLED
```

Comment:
I feel weird about the CSDK in this name. Can we call this `CONFIG_DEVICE_SHADOW_DEMO_ENABLED` instead?
```suggestion
CONFIG_DEVICE_SHADOW_DEMO_ENABLED
```

Reply:
sure